### DISCUSSION TITLE
Broken thumbnail image in media

### DISCUSSION THREAD

**Discussion by Pascualsolutions** [wintercms/winter#1017] [General]
Hello people :)
I have problem with broken thumbnail in media.
When i was on local ewerithing was fine but when i uploud to server i have broken image in media gallery.
![Screenshot from 2023-12-01 20-53-43](https://github.com/wintercms/winter/assets/75854186/fd9a6378-c4c7-4be2-a6d4-24a7f3ed75da)

Best regards,
Alex

**Comment by LukeTowers** [comment]
Your server is probably misconfigured and is eating requests to the image resizer endpoint.

**Reply by Pascualsolutions** [reply]
Thank you very much, i find solution. 
The problem is cloudflare, when i turn off some settings now work fine.
Best regards,
Alex

**Reply by LukeTowers** [reply]
@Pascualsolutions could you share which settings you had to modify to help anyone with similar issues in the future?

**Reply by Pascualsolutions** [reply]
Offcourse i turned off Rocket Loader.

**Reply by LukeTowers** [reply]
Hmm, do you have the CloudFlare plugin installed? https://github.com/wintercms/wn-cloudflare-plugin

Also, do you have any more information on how exactly Rocket Loader was causing the thumbnail images to break? I'd love to see if there's anything we can do out of the box to communicate to cloudflare to not do whatever it was that was breaking the resizer; all about streamlining the out of box experience as much as possible :)
